static/admin/config.yml

```diff
@@ -258,6 +258,69 @@ collections:
                         },
                     ],
             }
+          - {
+                label: "Editing Design",
+                name: "editing_design",
+                widget: "object",
+                required: false,
+                collapsed: true,
+                fields:
+                    [
+                        {
+                            label: "cRNA Target Site",
+                            name: "crna_target_site",
+                            widget: "string",
+                        },
+                        {
+                            label: "DNA Donor Sequence",
+                            name: "dna_donor_sequence",
+                            widget: "string",
+                        },
+                        { label: "Cas9", name: "cas9", widget: "string" },
+                        {
+                            label: "F Primer for PCR/Sequencing",
+                            name: "f_primer",
+                            widget: "string",
+                        },
+                        {
+                            label: "R primer for PCR/Sequencing",
+                            name: "r_primer",
+                            widget: "string",
+                        },
+                        {
+                            label: "Diagrams",
+                            name: "diagrams",
+                            widget: "list",
+                            collapsed: true,
+                            fields:
+                                [
+                                    {
+                                        label: "Diagram",
+                                        name: "diagram",
+                                        widget: "object",
+                                        fields:
```

Comment:
Any thoughts on whether we need a `diagram` to wrap each set of `title`, `image` and `caption`?

Reply:
Can you explain further? I see that in your PR the Diagram is a list, and ours is an object, which is how it was specified in the issue.

Reply:
Yeah sure, my question was whether, under the `diagrams`(plural) list, we need a `diagram`(singular) object to contain `title`, `image` and `caption`. 

Should we follow the structure in screenshot1 or screenshot2:
<img width="763" alt="Screenshot 2024-09-05 at 5 10 11 PM" src="https://github.com/user-attachments/assets/a04754ec-b8c8-4b2c-8757-e6c0d6ee1265">
<img width="834" alt="Screenshot 2024-09-05 at 2 51 21 PM" src="https://github.com/user-attachments/assets/35299efb-13b3-4d0d-a829-5ba6e04711a8">

Now that I'm looking at this ticket more closely, you're right - wrapping data in a `diagram` object makes more sense. 

@meganrm does screenshot1 look good to you? I can go ahead update the diagram part in our pr for consistency.

Reply:
the second one looks better to me. we should error on less nesting, both for the user, and it will make unpacking the data easier. I'm assuming that one is the list?

Reply:
Yes - the second one is a list of `Diagrams`. When `Add diagrams +` button is clicked, a new entry box is added to the list, which contains fields for `title`, `image` and `caption`, without nesting it within a `Diagram`.

Reply:
Ok changed this branch to match screenshot 2 and the way your PR looks. :)